abdiff/cli.py

```diff
@@ -72,7 +73,7 @@ def shared_job_options(cli_command: Callable) -> Callable:
         "--message",
         type=str,
         required=False,
-        help="Message to describe Job.",
+        help="Message to describe Job or Run.",
```

Comment:
Nice catch.  I'll update that to `shared_job_and_run_options`.  I wanted to avoid re-defining the CLI arguments `--job-directory` and `--message` when there were basically identical for creating new jobs and performing new runs.  But it did lead to a little awkwardness for naming.